gravitee-apim-rest-api/gravitee-apim-rest-api-management-v4/gravitee-apim-rest-api-management-v4-rest/src/main/resources/management-openapi-v4.yaml

```
@@ -1351,6 +1403,21 @@ components:
                     description: Plugin's version.
                     example: 1.0.0
                     minLength: 1
+        MoreInformation:
+            type: object
+            properties:
+                description:
+                    type: string
+                    description: Plugin's description.
+                    example: Let your consumers send synchronous, on-demand requests for events via a standard HTTP GET requests. Use a standard HTTP proxy to mediate between client applications as data consumers and event brokers such as Kafka.
+                documentationUrl:
+                    type: string
+                    description: Plugin's documentation URL.
+                    example: https://docs.gravitee.io
+                schemaImg:
+                    type: string
+                    description: Plugin's schema image as base64.
+                    example: data:image/png;base64,aFDqfda
```

I think we can use a field to specify the fact that it is a binary
```suggestion
                schemaImg:
                    type: string
                    format: base64
                    description: Plugin's schema image as base64.
                    example: data:image/png;base64,aFDqfda
```